pulpcore/cli/core/generic.py

```diff
@@ -21,6 +21,29 @@
 ##############################################################################
 # Generic reusable commands
 
+DATETIME_FORMATS = ["%Y-%m-%d", "%Y-%m-%dT%H:%M:%S"]
+
+
+def _task_filter_callback(
```

Comment:
we have `resource_option` but it didn't fit the bill.

Reply:
I was thinking more along the lines of a utility function that would encapsulate L30-45, that could be called with a path-suffix and then do the uuid-or-href computation, so it could be used by other, similar, options. As an example, task_group does something nearly identical here : https://github.com/pulp/pulp-cli/blob/main/pulpcore/cli/core/generic.py#L25

It would be nifty if, say , that code could devolve into a call to. say, `uuid_or_href(value, "task_groups")`, and this could become a call to `uuid_or_href(value, "tasks")`. Am I being clear here?

Reply:
Have a look again. The typical question here: Class or closure? (I made it a class this time for no particular preference...)